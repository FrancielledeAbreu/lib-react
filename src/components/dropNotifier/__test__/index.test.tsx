import React from 'react'
import renderer from 'react-test-renderer'
import DropNotifier from '../index'

describe('Snapshot', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<DropNotifier />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
