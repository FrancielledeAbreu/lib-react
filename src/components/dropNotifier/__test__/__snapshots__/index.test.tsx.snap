// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renders correctly 1`] = `
<div
  className="sc-bdnylx ubmVd"
>
  <div
    className="sc-gtssRu jafGCH"
  >
    <svg
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 512 512"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M256 48C141.6 48 48 141.601 48 256s93.6 208 208 208 208-93.601 208-208S370.4 48 256 48zm24 312h-48v-40h48v40zm0-88h-48V144h48v128z"
      />
    </svg>
  </div>
  <div
    className="sc-hKFyIo gbOhmc"
  />
  <div
    className="sc-dlnjPT irnZfw"
  >
    <svg
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 24 24"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M7,7 L17,17 M7,17 L17,7"
        fill="none"
        stroke="#000"
        strokeWidth="2"
      />
    </svg>
  </div>
</div>
`;
